falcon.install

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file
+ * Install and uninstall functions for the Falcon installation profile.
+ */
+
+/**
+ * Implements hook_install().
+ *
+ * Performs actions to set up the site for this profile.
+ */
+function falcon_install() {
+  // Create default Commerce Store.
+  // TODO: consider moving it to install task (hook_install_tasks()) with
+  // configurable store values?
+  $address = [
```

Comment:
Can we move store creation to `falcon_commerce` ? I believe it will make more sense.